### ISSUE TITLE
Sort by: Updated (newest or oldest first) – only sort on human updates

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#1365]
# Enhancement
Sort by: Updated (newest or oldest first) – only sort on human updates

Depends on #1364

## Context

Our Biblio team handles records with the "first in first out" principle. This means that the oldest records get reviewed first. Because of system updates, the team can no longer see this and has no control over their pipeline.
<img width="1391" alt="Screenshot 2024-07-12 at 15 14 46" src="https://github.com/user-attachments/assets/66a98006-11d6-4714-a013-71af6aa29b0a">

## Expected behaviour

We have three datapoints:
- Created by
  - Sorted option: Added (newest first)
  - Sorted option: Added (oldest first)
- System update (if available)
  - [ ] _No sort_
- Edited by
  - [ ] Sort option: Updated (newest first)
  - [ ] Sorted option: Updated (oldest first) (see #1364)

<img width="947" alt="Screenshot 2024-02-05 at 09 51 08" src="https://github.com/ugent-library/biblio-backoffice/assets/1572885/64dde240-cf32-4022-ae99-e9fa92042358">

## Out of scope

Making sure we see the date and hour instead of "3 days ago".

**Comment by mietcls** [comment]
Keeping this for after our new set-up. Not urgent now.
